tests/app/questionnaire/test_value_source_resolver.py

```diff
@@ -534,6 +534,57 @@ def test_calculated_summary_value_source(mocker, list_item_id):
     )
 
 
+@pytest.mark.parametrize(
+    "list_item_id",
+    [None, "item-1"],
+)
+def test_new_calculated_summary_value_source(mocker, list_item_id):
+    schema = mocker.MagicMock()
+    schema.get_block = Mock(
+        return_value={
+            "id": "number-total",
+            "type": "CalculatedSummary",
+            "calculation": {
+                "operation": {
+                    "+": [
+                        {"source": "answers", "identifier": "number-answer-1"},
+                        {"source": "answers", "identifier": "number-answer-2"},
+                    ]
```

Comment:
The new operator should be tested as its own thing also. Same as if we were to add any other new operators. Need to test a recursive one also.

Reply:
Add tests for the operator and a recursive test 👍